src/wrappers/PhyxWrapper.js

```diff
@@ -37,31 +37,62 @@ class PhyxWrapper {
    *    2. We have to convert phylogenies into OWL restrictions.
    *    3. Insert all matches between taxonomic units in this file.
    *
-   * @param {string} [baseURI="#"] - The base URI to use when generating this Phyx document.
+   * @param {string} [baseURI=""] - The base URI to use when generating this Phyx document.
+   *    We will add a '#' if needed, so don't add it yourself.
```

Comment:
As stated, this seems self-contradictory. `We will add a '#' if needed` would mean that if I already added a `#`, it would not be added again, because then adding it is not needed. So either adding a (trailing, presumably) `#` by the caller is harmless, or the method will _always_ add a `#` (so if the trailing character is `/` you will still add a `#`?) whether it's needed or not, but not both.

Reply:
Good point! I've cleaned it all up in 9dc1268 and improved the documentation in 8a64a8a. The new behavior is:
 - If baseURI is left blank (i.e. set to `''`), then all IDs in the Phyx file will be unchanged, and relative IDs will be generated as needed (e.g. `#phylogeny0`).
 - If baseURI is set, we expect it to end with a `#` or a `/`, and we will generate absolute IDs using the baseURI as needed (e.g. setting baseURI to `http://example.org/` will cause e.g. `http://example.org/phylogeny0` to be generated).
- Note that if baseURI is set, then relative IDs already in the file that begin with `#` (e.g. `#Crocodylidae`) will be turned into absolute IDs (e.g. `http://example.org/Crocodylidae`). The JSON-LD reader I'm using for testing in this PR appears to require this, so I suspect that other JSON-LD readers might have this requirement. 

Let me know if that seems clear; otherwise, I'll take another stab at improving it!